style.css

```diff
@@ -0,0 +1,157 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
```

Comment:
- Good Job implementing changes requested from your last reviewer, but there are still some issues with the page, it's not filling the screen as expected and there are some white spaces around. Kindly fix that

![Screenshot from 2023-04-28 13-44-30](https://user-images.githubusercontent.com/97834160/235140215-b4199605-26a7-4403-934b-13194a1c6536.png)